fluentd/lib/fluent-plugin-collected/Gemfile.lock

```diff
@@ -1,36 +1,34 @@
 PATH
   remote: .
   specs:
-    fluent-plugin-collected (1.0.0)
-      fluentd (>= 0.14.20, < 2)
-      prometheus-client (< 0.10)
+    fluent-plugin-collected (1.1.0)
+      fluentd (= 1.14.5)
+      prometheus-client (>= 2.1.0)
 
 GEM
   remote: https://rubygems.org/
   specs:
-    concurrent-ruby (1.1.9)
+    concurrent-ruby (1.1.10)
     cool.io (1.7.1)
-    fluent-plugin-prometheus (1.8.5)
+    fluent-plugin-prometheus (2.0.2)
       fluentd (>= 1.9.1, < 2)
```

Comment:
This plugin is updated, can we also set fluentd to 1.14.5 here?

Reply:
wont matter her as the lock pegs it to 1.14.5 later in the lock file

Reply:
I did set fluentd to 1.14.5 everywhere just to be clear.